cypress/e2e/tests/pages/charts/monitoring-istio.spec.ts

```diff
@@ -46,18 +41,30 @@ describe('Charts', { tags: ['@charts', '@adminUser'] }, () => {
         terminal.closeTerminal();
       });
 
-      // Don't actually install the chart, just navigate to the edit options page
       beforeEach(() => {
-        cy.login();
-        chartsPage.goTo();
+        cy.getRancherResource('v1', 'catalog.cattle.io.clusterrepos', 'rancher-charts?link=index', 200).then((resp: Cypress.Response<any>) => {
+          const version = resp.body.entries['rancher-monitoring-crd'][1]['version'];
```

Comment:
This should be `rancher-monitoring`, rather than `-crd`. 

I really like this double checking of the automatically selected chart version against the latest from api response... BUT here we're not checking the first entry but the second. Guessing you're testing on a system below 2.9.0 which means the first entry is not presented in the UI (each chart has annotations that restrict the visibility, the first one will only show if rancher is 2.9.0+... so once 2.9.0 is released this test will break).

In pains me to say but i think we shouldn't check the version.

Reply:
Roger that, will remove the check. thanks for catching that